### PR TITLE
Collaborating: update context to cats-as-data

### PR DESCRIPTION
**Pull request by mcritchlow**
Closes: #22 

Proposed Changes:
- Have Collaborators update based on the README todo list
- Replace pushing directly to master with a branch+PR workflow
- Require Owner to merge the Collaborator's PR
- Remove quiz/question that reinforces the idea of pushing directly to
master as a Collaborator

### CODE REVIEW

File: _episodes/10-collaborating.md [hunk 1]
```diff
@@ -39,45 +40,59 @@ Next, the Collaborator needs to download a copy of the Owner's repository to her
 her `Desktop` folder, the Collaborator enters:
 
 ~~~
-$ git clone https://github.com/vlad/planets.git ~/Desktop/vlad-planets
+$ git clone https://github.com/ccline/cats-as-data.git ~/Desktop/ccline-cats-as-data
 ~~~
 {: .bash}
 
-Replace 'vlad' with the Owner's username.
+Replace 'ccline' with the Owner's username. Also note the username prefix on the directory name. This is done because
```
Comment (dunn) [inline _episodes/10-collaborating.md]: Nit: backticks around `ccline`?

File: _episodes/10-collaborating.md [hunk 2]
```diff
@@ -39,45 +40,59 @@ Next, the Collaborator needs to download a copy of the Owner's repository to her
...
+you already have your own project named `cats-as-data` in the Desktop folder.
 
 ![After Creating Clone of Repository](../fig/github-collaboration.svg)
 
 The Collaborator can now make a change in her clone of the Owner's repository,
-exactly the same way as we've been doing before:
+exactly the same way as we've been doing before.
+Please choose one of the tasks from the cleanup plan created in the Branching lesson.
```
Comment (dunn) [inline _episodes/10-collaborating.md]: Is this supposed to be a new paragraph?

File: _episodes/10-collaborating.md [hunk 3]
```diff
@@ -87,20 +102,40 @@ Delta compression using up to 4 threads.
...
 
-To download the Collaborator's changes from GitHub, the Owner now enters:
+Notice that GitHub will automatically select the branch to merge your Pull Request into. This is usually `master` and
+exactly what you want. However, you can change this to any existing branch in the repository.
+
+The Collaborator's Pull Request will allow the Owner to:
+- review and accept the proposed changes
+- perform a merge, just like we did in the Merging branches lesson
+
+The Owner should now review the Pull Request and [Merge](https://help.github.com/articles/merging-a-pull-request/) it.
```
Comment (dunn) [inline _episodes/10-collaborating.md]: Do all these terms need to be capitalized?

File: _episodes/10-collaborating.md (hunk 1 above)
Comment (mcritchlow) [reply _episodes/10-collaborating.md]: You're really ticing me off. yes fine, that is better :)

File: _episodes/10-collaborating.md (hunk 2 above)
Comment (mcritchlow) [reply _episodes/10-collaborating.md]: Oh good catch, ty

File: _episodes/10-collaborating.md (hunk 3 above)
Comment (mcritchlow) [reply _episodes/10-collaborating.md]: I'm keeping with the Owner/Collaborator pattern they have. I do think it helps make it a bit more obvious who is supposed to do what. but no don't think Pull Request and Merge need to be. I'll clean that up. Thanks.

File: _episodes/10-collaborating.md [hunk 4]
```diff
@@ -39,45 +40,58 @@ Next, the Collaborator needs to download a copy of the Owner's repository to her
...
-$ git clone https://github.com/vlad/planets.git ~/Desktop/vlad-planets
+$ git clone https://github.com/ccline/cats-as-data.git ~/Desktop/ccline-cats-as-data
 ~~~
 {: .bash}
 
-Replace 'vlad' with the Owner's username.
+Replace `ccline` with the Owner's username. Also note the username prefix on the directory name. This is done because
+you already have your own project named `cats-as-data` in the Desktop folder.
 
 ![After Creating Clone of Repository](../fig/github-collaboration.svg)
```
Comment (dunn) [inline _episodes/10-collaborating.md:50]: I guess we can't avoid the old repo name / user name in this one?

Comment (dunn) [reply _episodes/10-collaborating.md:50]: Actually it's SVG so maybe we can 😈

Comment (mcritchlow) [reply _episodes/10-collaborating.md:50]: brilliant! hacking it now.

Comment (mcritchlow) [reply _episodes/10-collaborating.md:50]: better? https://github.com/ucsdlib/git-novice/blob/0446c9ac58304e6846843ce53b99e38bfe9307a2/fig/github-collaboration.svg

Also took the opportunity to update it with a more accurate path to reflect what we're having them do.

Comment (dunn) [reply _episodes/10-collaborating.md:50]: yesssss

**Review by dunn — APPROVED** [review]

**Comment by mcritchlow** [comment]
@chrissyrissmeyer look ok to you?

**Comment by chrissyrissmeyer** [comment]
@mcritchlow it is great!

**Review by chrissyrissmeyer — APPROVED** [review]

**Pull request merged**